.yarnrc.yml

```
@@ -6,4 +6,15 @@ plugins:
     - path: .yarn/plugins/@yarnpkg/plugin-version.cjs
       spec: '@yarnpkg/plugin-version'
 
+supportedArchitectures:
+  cpu:
+    - arm
+    - arm64
+    - x64
+    - ia32
+  os:
+    - darwin
+    - linux
+    - win32
```

Looking at [the docs for this](https://yarnpkg.com/configuration/yarnrc#supportedArchitectures), I think we should include `current` in both to allow yarn to install packages on less usual architectures, if needed.

Once that is done we can probably remove some of the more unusual cpu types I don't think we need to install the 32bit versions of packages for example.

What do you think of:

```suggestion
supportedArchitectures:
  cpu:
    - current
    - arm64
    - x64
  os:
    - current  
    - darwin
    - linux
    - win32
```